boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst

```diff
@@ -0,0 +1,169 @@
+.. _nrf52_ruuvi:
+
+nRF52-ruuvi
+##############
+
+Overview
+********
+
+RuuviTag is a generic sensor node, that fits perfectly with many uses.
+
+Its open-source design enables various customisation possibilities. Both the hardware and software are 100% open.
+
+.. figure:: img/ruuvitag-pcb.jpg
+     :width: 442px
+     :align: center
+     :alt: RUUVI Tag
+
+     RUUVI Tag
+
+More information about the board can be found at the
+`ruuvitag website`_.
+
+Hardware
+********
+
+RuuviTag is built from the ground up using the latest technology available.
+
+* Nordic Semiconductor nRF52832 System-on-Chip
+* STMicroelectronics LIS2DH12 accelerometer
+* Bosch BME 280 temperature + relative air humidity + air pressure sensor
+* NFC™-A tag antenna
+* 1000mAh CR2477 battery
+* Recommended -20ºC to +65ºC (max -40ºC to +85ºC)
+* 2 buttons
+* 2 LEDs
+* 45 mm diameter (PCB)
+* 52 mm diameter (enclosure)
+* Long range RF antenna
+
+`ruuvitag datasheet`_
+
+Supported Features
+==================
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| NVIC      | on-chip    | nested vectored      |
+|           |            | interrupt controller |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial port          |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTT       | on-chip    | console              |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF52 DK website`_ and `Nordic Semiconductor Infocenter`_
```

Comment:
You're missing the definitions of these two link targets.  At the bottom of this file add them with:

    .. _nrf52 DK website:  url to site
    .. _Nordic Semiconductor Infocenter: url to site